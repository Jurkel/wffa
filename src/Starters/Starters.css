.starter-list {
  margin-left: 35px;
}


.starters-wrapper  h4 {
  color: #FFD08A;
  font-size: 16px;
  font-family: 'Courier';
}

.starters-wrapper h3 {
  color: #f2f2f2;
  font-size: 16px;
  margin-left: 10px;
  margin-top: 20px;
}

.starter-flex {
  display: flex;
  flex-direction: row;
  margin-bottom: -25px;
}

.starters-team {
  text-align: right;
  font-weight: 500;
  color: #FFD08A;
}