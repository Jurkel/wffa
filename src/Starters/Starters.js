import React from 'react'
import './Starters.css'

class Starters extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      starters: []
    }
  }

  // componentDidMount() {
  //   console.log('starter data 1 => ' + JSON.stringify(this.props.rosterData));
  //   const starterArray = this.props.rosterData.starters;
  //   console.log('starter array 1 => ' + JSON.stringify(starterArray))
  //   this.setState({
  //     starters: starterArray
  //   }, () => {
  //     console.log('starter array set state => ' + this.state.starters)
  //   })
  // }

  render() {

    console.log('This is render from starters' + this.props.starters)
    // const starters = this.props.starters;
    let starters = this.props.starters.map((list, key) => 
      <div 
      key={key}
      className='starter-flex'>
        <h4>{list}</h4>
      </div>
    )
  
  
  return (
    <div className='starters-wrapper'>
      <h2 className='player-title'>Starters</h2>
      <div className='starter-list'>
        {starters}
      </div>
    </div>
  )
  }
}

export default Starters;