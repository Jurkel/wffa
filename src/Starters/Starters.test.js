import React from "react";
import { shallow } from "enzyme";
import Starters from "./Starters";

it("renders Starters component", () => {
  const mockProps = {
    starters: [
      {
        player: {
          position: "",
          full_name: "",
          team: "",
        },
      },
    ],
  };
  expect(shallow(<Starters {...mockProps} />)).toMatchSnapshot();
});