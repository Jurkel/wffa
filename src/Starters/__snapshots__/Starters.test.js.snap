// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Starters component 1`] = `ShallowWrapper {}`;
